app/assets/javascripts/converse.js

```diff
@@ -0,0 +1,23 @@
+//= require converse.js.js
+
+// initialize converse xmpp client
+$(document).ready(function() {
+  if (app.currentUser.authenticated()) {
+    $.post("/user/auth_token", null, function(data) {
+      if (converse && data.token) {
```

Comment:
I guess I can ignore that. If someone points me to the right exception file for pronto I will add it.